tests/test-initd-scripts.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
+
+  vars:
+    jenkins_initd_files:
+      - "{{ role_path }}/tests/README.md"
```

Comment:
Is it possible to add in any more meaningful / practical tests here? Something that would be an actual init script that would affect Jenkins, to make sure that if we copy one in, Jenkins actually boots and uses it and doesn't fail?

Reply:
I think this outstanding review change request has been made?